### ISSUE TITLE
[A11y]: Make panel header not overlay main header

### ISSUE THREAD

**Issue by ethanwinters** [carbon-design-system/carbon-ai-chat#601]
### Package version

1.1.0

### Automated testing tool and ruleset

None

### Description

Per a11y review, we should be opening the panels to underneath the chat header instead of on top of it.

### Reproduction / example

none

### Steps to reproduce

From demo site select "buttons". Click on "open a panel". The panel goes over the header. The minimize/close button is unclear if its for the panel or for the whole chat.

### Suggested severity

None

### Code of conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-ai-chat/issues) for duplicate problems
